From 9473c3820bec64897e6e51159d0ccfe45f7b8ebf Mon Sep 17 00:00:00 2001
From: Will Thompson <wjt@endlessos.org>
Date: Wed, 27 Apr 2022 22:00:32 +0100
Subject: [PATCH 05/13] metainfo: Add bugtracker URL

I could not locate appropriate URLs for the other types documented at
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url,
such as a translation link, that were not redundant with this.
---
 gobby-0.5.metainfo.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gobby-0.5.metainfo.xml b/gobby-0.5.metainfo.xml
index f25ed91..a89d5d8 100644
--- a/gobby-0.5.metainfo.xml
+++ b/gobby-0.5.metainfo.xml
@@ -27,4 +27,5 @@
   </screenshots>
 
   <url type="homepage">https://gobby.github.io/</url>
+  <url type="bugtracker">https://github.com/gobby/gobby/issues</url>
 </component>
-- 
2.30.2

